src/libs/ajax/Metrics.ts

```diff
@@ -55,11 +56,11 @@ export const Metrics = (signal?: AbortSignal) => {
 
     syncProfile: withErrorIgnoring(() => {
       return fetchBard('api/syncProfile', _.merge(authOpts(), { signal, method: 'POST' }));
-    }),
+    }) as () => Promise<void>,
```

Comment:
These functions were typed as `any` because `withErrorIgnoring` is typed as `any`.